packages/apps-config/src/api/spec/subsocial.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+import * as subsocialDefinitions from "@subsocial/types/substrate/interfaces/definitions";
```

Comment:
I believe the linter would complain here (have not looked, but there are CI failures, so assuming this is one of them)